src/styles/Home.module.css

```diff
@@ -3,7 +3,7 @@
 }
 
 /* NON SESSION USER HOMEPAGE */
-.greetwrap {
+.greet_wrap {
```

Comment:
Let's go with camel case for CSS modules as recommended here: https://github.com/css-modules/css-modules#naming